packages/repo-tools/package.json

```diff
@@ -33,17 +33,22 @@
     "@backstage/cli-common": "workspace:^",
     "@backstage/errors": "workspace:^",
     "@manypkg/get-packages": "^1.1.3",
-    "@microsoft/api-documenter": "^7.17.11",
-    "@microsoft/api-extractor": "^7.23.0",
-    "@microsoft/api-extractor-model": "^7.17.2",
-    "@microsoft/tsdoc": "0.14.1",
+    "@microsoft/api-documenter": "^7.19.26",
+    "@microsoft/api-extractor": "^7.33.7",
+    "@microsoft/api-extractor-model": "^7.25.3",
+    "@microsoft/tsdoc": "^0.14.2",
+    "@microsoft/tsdoc-config": "~0.16.1",
+    "@rushstack/node-core-library": "^3.53.3",
+    "@spotify/prettier-config": "^14.1.3",
     "chalk": "^4.0.0",
     "commander": "^9.1.0",
     "fs-extra": "10.1.0",
     "glob": "^8.0.3",
     "is-glob": "^4.0.3",
     "minimatch": "^5.1.1",
-    "ts-node": "^10.0.0"
+    "prettier": "^2.8.1",
+    "ts-node": "^10.0.0",
+    "typescript": "^4.9.4"
```

Comment:
It might make the most sense to make this a peer dependency, but not sure. I have a feeling 4.9 will break this repo for example, because we don't support it yet. Then again it might not be used for anything too important. I'd default to a peer dep with a pretty broad range, and then switch it up if that doesn't work well.

Reply:
yes, giving this a try